R/plot_variance.R

```diff
@@ -0,0 +1,66 @@
+#' Plot as a heatmap the scaled between imputation variance for every cell
+#'
+#' This function plots the cell-level between imputation variance. The function
+#' scales the variances column-wise and plots the data image as a heatmap.
+#' Darker red cells indicate more variance, lighter cells indicate less variance.
+#' white cells represent observed cells or unobserved cells with zero between
+#' imputation variance.
+#'
+#' @param object A `mice` generated multiply imputed data set of class `mids`.
+#'
+#' @return An object of class `ggplot`.
+#' @importFrom dplyr mutate
+#' @importFrom dplyr across
+#' @importFrom dplyr everything
+#' @importFrom ggplot2 aes
+#' @importFrom mice mdc
+#' @importFrom dplyr select
+#' @importFrom dplyr group_by
+#' @importFrom dplyr summarise
+#' @importFrom dplyr ungroup
+#' @importFrom tidyr pivot_longer
+#' @importFrom stats var
```

Comment:
See comment above about importing functions. The `ggmice` package sticks with the first option recommended here: https://r-pkgs.org/namespace.html#import-r

Reply:
I agree with the readability argument. However, since these are such commonly used functions: wouldn't it be much more efficient in the long run to define these imports? That would avoid `::` your way into oblivion and it would avoid unnecessary repetitions of e.g. `dplyr::mutate()` and `ggplot2::aes()` across and within function. Plus, it's faster.